src/components/skip-link/template.njk

```diff
@@ -0,0 +1,3 @@
+<a class="govuk-c-skip-link{%- if params.classes %} {{ params.classes }}{% endif -%}"{%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %} href="{%- if params.href %}{{ params.href }}{% else %}#content{% endif -%}">
```

Comment:
It's possibly a bit pedantic to call out argument ordering, but I'd usually expect `href` to be the first argument for a link – worth moving it to the front?

Could also simplify the href logic to just `{{ params.href if params.href else '#content' }}` or maybe even just `{{ params.href | default('#content') }}`